.ci/restart_kubernetes.sh

```diff
@@ -25,4 +25,4 @@ kubectl rollout status --namespace=${JOB_NAME} deployment/microservice-product-s
 PHP_FPM_POD=$(kubectl get pods -n ${JOB_NAME} -l app=webserver-php-fpm -o=jsonpath='{.items[0].metadata.name}')
 
 # Run phing build in the pod
-kubectl exec --namespace=${JOB_NAME} ${PHP_FPM_POD} ./phing db-create test-db-create build-demo-dev-quick
+kubectl exec --namespace=${JOB_NAME} ${PHP_FPM_POD} ./phing db-create test-db-create error-pages-generate build-demo-dev-quick
```

Comment:
```suggestion
kubectl exec --namespace=${JOB_NAME} ${PHP_FPM_POD} ./phing db-create test-db-create build-demo-dev-quick error-pages-generate 
```
there is problem with empty database after task delete namespaces is executed

Reply:
That makes sense :+1: